skuba-update/test/os/suse/install_skuba_update.sh

```diff
@@ -19,15 +19,15 @@ set -xe
 release=$(head -n 1 /etc/os-release)
 if [ "$release" = "NAME=\"SLES\"" ]; then
     rpm -e container-suseconnect
-    zypper ar --no-gpgcheck http://download.opensuse.org/distribution/leap/15.1/repo/oss/ leap15.1
-    zypper ar --no-gpgcheck http://download.opensuse.org/update/leap/15.1/oss updates
+    zypper ar --no-gpgcheck https://download.opensuse.org/distribution/leap/15.2/repo/oss/ leap15.2
+    zypper ar --no-gpgcheck https://download.opensuse.org/update/leap/15.2/oss updates
     zypper ref
 fi
 
 zypper in -y python3-setuptools lsof
 
 if [ "$release" = "NAME=\"SLES\"" ]; then
-    zypper rr leap15.1
+    zypper rr leap15.2
```

Comment:
one quick little nit. this really seems like a unrelated change. in my opinion this would go in another PR, or at least another commit to help with debugging later.

Reply:
I removed this part of the change.